dataView/DataViewModel.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {StoreSelectionModel} from 'hoist/data';
+import {GridContextMenu} from 'hoist/grid/GridContextMenu';
+
+/**
+ * DataViewModel is a wrapper around GridModel, which shows sorted data in
+ * a single column, using a defined component for rendering each item.
+ */
+export class DataViewModel {
+
+    // Immutable public properties
+    itemFactory = null;
+    store = null;
+    selection = null;
+    contextMenuFn = null;
+
+    static defaultContextMenu = () => {
+        return new GridContextMenu([
+            'copy',
```

Comment:
Weird for this thing to be using a "GridContextMenu"

Reply:
Yeah, I thought the same - but the GridContextMenu has all built in support for 'requireRecords' etc, so seemed a bit wasteful to make DataViewContextMenu that would be basically be identical